src/Disks/ObjectStorages/S3/S3ObjectStorage.cpp

```diff
@@ -248,13 +250,18 @@ std::unique_ptr<WriteBufferFromFileBase> S3ObjectStorage::writeObject( /// NOLIN
         scheduler = threadPoolCallbackRunner<void>(getThreadPoolWriter(), "VFSWrite");
 
     auto clients_ = clients.get();
+
+    BlobStorageLogWriter blob_storage_log = getBlobStorageLog();
```

Comment:
Should there be a way to enable/disable writing to blob_storage_log with a query/profile setting?

Reply:
I don't think it make lot's of sense, since user can configure it in server level config. In that case it can be used to debug some issues connected with missing or unexpected blobs. If logging is disabled for half of the requests it will not be helpful anymore, so probably it may be just disabled in config.